src/NuGet.Core/NuGet.ProjectModel/Strings.resx

```diff
@@ -152,4 +152,58 @@
   <data name="MissingVersionOnTool" xml:space="preserve">
     <value>Tools must specify a version range.</value>
   </data>
+  <data name="PackagesLockFile_CentralPackageVersionRemoved" xml:space="preserve">
+    <value>Central package management file(s) doesn't contain version range for {0} package which is specified as CentralTransitive dependency in the lock file.</value>
+  </data>
+  <data name="PackagesLockFile_IncompatibleLockFileVersion" xml:space="preserve">
+    <value>Version specified in the packages.lock.json file is incompatible. Current tooling supports version {0} and previous formats.</value>
+  </data>
+  <data name="PackagesLockFile_MismatchedTargetFrameworks" xml:space="preserve">
+    <value>The project target frameworks are different than the lock file's target frameworks. Lock file target frameworks: {0}. Project target frameworks {1}.</value>
+    <comment>0 - Semicolon delimited list of lock file target frameworks, 1 - semicolon delimited list of lock file target frameworks</comment>
+  </data>
+  <data name="PackagesLockFile_NewTargetFramework" xml:space="preserve">
+    <value>The project target framework {0} was not found in the lock file.</value>
+    <comment>0 - project target framework</comment>
+  </data>
+  <data name="PackagesLockFile_None" xml:space="preserve">
```

Comment:
nit: This is a generic `None`. 
So you can change the name of the resource to None.

Reply:
changed the resource key to "None".